frontend/src/lib/components/apps/editor/ComponentEditor.svelte

```diff
@@ -42,11 +42,6 @@
 	{/if}
 
 	<div
-		on:pointerdown={(e) => {
-			if ($mode === 'preview') {
-				e?.stopPropagation()
-			}
-		}}
```

Comment:
Why delete this ? This is to avoid conflict with svelte-grid onpointerdown

Reply:
I apply styles to wrapper elements based on which element is clicked, and because of this, the event would never reach the wrapper in preview mode. I consulted with @fatonramadani and tested things but nothing weird came up. Do you have an example where this would cause issues?